tar.go

```diff
@@ -338,7 +340,16 @@ func (t *Tar) Write(f File) error {
 		return fmt.Errorf("missing file name")
 	}
 
-	hdr, err := tar.FileInfoHeader(f, f.Name())
+	filename := f.Name()
```

Comment:
FYI, previous commits in this change had left the `filename` (previously called `linkTarget`) empty for non-symlinks, which I _think_ was probably a bug. Should be fixed now.

Reply:
Oh, you're right. Pushed a commit that fixes that. Thanks. Am ready to merge if you are!

Reply:
> Am ready to merge if you are!

Sure, go ahead and let's close a bunch of open issues! 😄 

I assume you will take care of filing issues for things discovered in this PR, if you feel they should be addressed at some point:

* Add option to dereference symlinks before adding to archive

* Add test coverage for `Extract` method

* (maybe) Remove vestigial hardlink support

* (maybe) Investigate weird permissions issue with ZIP files created on Windows

Reply:
Yep, I think those are all suitable for other issues and PRs. Thanks again!